p2p_wallet/Scenes/Main/Sell/SellCoordinator.swift

```diff
@@ -148,6 +150,31 @@ final class SellCoordinator: Coordinator<SellCoordinatorResult> {
                 .handleEvents(receiveCompletion: { [unowned self] _ in
                     self.viewModel.warmUp()
                 }).eraseToAnyPublisher()
+
+        case .moonpayInfo:
+            moonpayInfoViewController = UIHostingController(
```

Comment:
I would have extracted it in separate coordinator to weight off this file but whatever) it is not critical

Reply:
didn't want to make one more coordinator that doesn't do any logic, only shows screen